core/src/test/java/com/linecorp/armeria/server/websocket/DelegatingWebSocketServiceTest.java

```diff
@@ -27,6 +27,7 @@
 import org.reactivestreams.Subscription;
 
 import com.linecorp.armeria.client.BlockingWebClient;
+import com.linecorp.armeria.client.WebClient;
```

Comment:
```suggestion
```

Reply:
https://github.com/line/armeria/pull/5590/commits/95592de2d319941bb4b2484b0cbd7b3861dda5d9 oh I simply remove this change~! 🙇